### PR TITLE
fix: return webpush style response for bridged users

### LINKED ISSUE #651: 200 OK response from Autopush
When sending a message, Autopush responds with 200 instead of 201 and a "Message Sent".

 curl -v "https://updates.push.services.mozilla.com/wpush/v1/gAAAAA..." --request POST --header "TTL: 60" --header "Content-Length: 0"

### PR DESCRIPTION
**Pull request by bbangert**
Closes #651

### CODE REVIEW

File: autopush/router/gcm.py [hunk 1]
```diff
@@ -181,4 +183,9 @@ def _process_reply(self, reply, uaid_data):
 
         self.metrics.increment("updates.client.bridge.gcm.succeeded",
                                self._base_tags)
-        return RouterResponse(status_code=200, response_body="Message Sent")
+        location = "%s/m/%s" % (self.ap_settings.endpoint_url,
+                                notification.version)
+        return RouterResponse(status_code=201, response_body="",
+                              headers={"TTL": ttl,
+                                       "Location": location},
+                              logged_status=200)
```
Comment (jrconlin) [inline autopush/router/gcm.py]: Shouldn't this be "201"?

Comment (bbangert) [inline autopush/router/gcm.py]: logged_status can be either 200 or 202 for how we log.

**Comment by jrconlin** [comment]
Filed https://github.com/mozilla-services/autopush/issues/653 to track fix for zero padding for months.

**Comment by jrconlin** [comment]
It'd be nice to have tests to make sure we don't drop the return values, but not a show stopper for this patch.

r+

**Comment by pjenvey** [comment]
r+

**Pull request merged**